tools/githubChangelogProcessor.php

```diff
@@ -0,0 +1,347 @@
+<?php
+/*
+ *	Github webhook In-game PR Announcer and SQL Changelog Generator for ParadiseSS13
+ *  Modified for Paradise by: AffectedArc07
+ *	Original code by: MrStonedOne
+ *
+ *	For documentation on the changelog generator see https://tgstation13.org/phpBB/viewtopic.php?f=5&t=5157
+ *	To hide prs from being announced in game, place a [s] in front of the title
+ *	All runtime errors are echo'ed to the webhook's logs in github
+ */
+
+/**CREDITS:
+ * GitHub webhook handler template.
+ *
+ * @see  https://developer.github.com/webhooks/
+ * @author  Miloslav Hula (https://github.com/milo)
+ */
+
+
+//CONFIG START (all defaults are random examples, do change them)
+//Use single quotes for config options that are strings.
+
+//Github lets you have it sign the message with a secret that you can validate. This prevents people from faking events.
+//This var should match the secret you configured for this webhook on github.
+//This is required as otherwise somebody could trick the script into leaking variables.
+$hookSecret = '08ajh0qj93209qj90jfq932j32r';
+
+$dbServer = "localhost"; // Hostname of the database server (default localhost)
+$dbUser = "root"; // Database username (default root)
```

Comment:
This requires a comment describing the permissions that the user needs. I assume it only needs all permissions to one table, 'changelog', and no permissions to any other table.